pkg/lib/cockpit-components-privileged.jsx

```diff
@@ -47,42 +49,23 @@ export function Privileged({ excuse, allowed, placement, tooltipId, children })
 /**
  * Convenience element for a Privilege wrapped Button
  */
-export class PrivilegedButton extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = { allowed: !!this.props.permission.allowed };
-        this.onPermissionChanged = this.onPermissionChanged.bind(this);
-    }
-
-    componentDidMount() {
-        this.props.permission.addEventListener("changed", this.onPermissionChanged);
-        this.onPermissionChanged();
-    }
-
-    componentWillUnmount() {
-        this.props.permission.removeEventListener("changed", this.onPermissionChanged);
-    }
+export const PrivilegedButton = ({ tooltipId, placement, excuse, buttonId, onClick, ariaLabel, variant, children, permission }) => {
```

Comment:
The `permission` property is not passed, declared, or used any more. Please drop from the list.

Would also be nice to move this commit first, so that the previous commit doesn't introduce something broken.